File: Java/Area of a Circle/areaofcircle.java
import java.util.Scanner;
import java.text.DecimalFormat;

public class areaofcircle
{
    public static void main(String[] args) {
    
        Scanner sc = new Scanner(System.in);
        
        double area;
        double radius;
        
        System.out.print("Enter a radius (in cm): ");
        
        radius = sc.nextDouble();          
        area =  Math.PI * Math.pow(radius, 2);
        
        DecimalFormat dec = new DecimalFormat("#.##");
        String area_1 = dec.format(area);
        
        System.out.println("The radius of " + radius + " cm is " + area_1 + " cm².");
    }
}

File: C++/Area of a Circle/areaofcircle.cpp
#include <iostream>
#include <cmath>
#include <iomanip>
#include "areaofcircle.h" 

int main() {

    double radius = getInput<double>("Enter the radius of the circle in cm: ");    
    
    while(radius <= 0){
        std::cout<<"\nRadius can't be "<<radius<<"cm. Try again.\n"<<std::endl;
        radius = getInput<double>("Enter the radius of the circle in cm: ");
    }
       
    double area = M_PI * std::pow(radius, 2);
    
    std::cout<<"\nThe area of a circle of radius "<<radius<<"cm is "<<std::fixed<<std::setprecision(2)<<area<<"cm²";
    
    return 0;
}